### ISSUE TITLE
Is the dataset size 1000(images) too small for both LLaVA-NeXT and OneVision?

### ISSUE THREAD

**Issue by Bleking** [LLaVA-VL/LLaVA-NeXT#351]
Hi. I have been trying to finetune LLaVA-NeXT and OneVision with my custom dataset with 1,000 images for training; there are 200 images for validation. I have had no issues with training when using LLaVA-v1.5 and v1.6 but things seem different with NeXT and OneVision since the loss seems to converge too fast before 1 epoch is over.

Therefore, I would like to check if 1000 size is way too small for these models, in general, unlike the previous versions, v1.5 and v1.6.

I will provide you with the scripts if you need some context.

Thank you.
